libraries/rush-sdk/src/index.ts

```diff
@@ -226,14 +240,4 @@ for (const property in sdkContext.rushLibModule) {
   }
 }
 
-/**
- * Used by the .js stubs for path-based imports of `@microsoft/rush-lib` internal APIs.
- */
-export function _rushSdk_loadInternalModule(srcImportPath: string): unknown {
-  if (!exports._RushInternals) {
-    throw new Error(
-      `Rush version ${exports.Rush.version} does not support internal API imports via rush-sdk`
-    );
-  }
-  return exports._RushInternals.loadModule(srcImportPath);
-}
+export default exports;
```

Comment:
This is a severe breaking change. We import this module using named imports and/or a namespace import, not a default import.

Though I guess the webpack run fixes this?